src/coverlet.core/Helpers/SourceRootTranslator.cs

```diff
@@ -94,9 +96,11 @@ public SourceRootTranslator(string moduleTestPath, ILogger logger, IFileSystem f
           _logger.LogWarning($"Malformed mapping '{mappingRecord}'");
           continue;
         }
+#pragma warning disable IDE0057 // Use range operator
         string projectPath = mappingRecord.Substring(0, projectFileSeparatorIndex);
         string originalPath = mappingRecord.Substring(projectFileSeparatorIndex + 1, pathMappingSeparatorIndex - projectFileSeparatorIndex - 1);
         string mappedPath = mappingRecord.Substring(pathMappingSeparatorIndex + 1);
+#pragma warning restore IDE0057 // Use range operator
```

Comment:
I see disabling this warning in a couple of files. Can we adapt `.editorconfig` instead?

Reply:
Yes, this could be done but this will hide also new findings. The range operators are pretty need and with the current source suppression this is easy to find and could be resolved in another PR. This one is already huge.